packages/react-query/src/__tests__/suspense.test.tsx

```diff
@@ -669,7 +669,7 @@ describe("useQuery's in Suspense mode", () => {
         {
           retry: false,
           suspense: true,
-          useErrorBoundary: (err) => err !== 'Local Error',
+          useErrorBoundary: (err) => err.message !== 'Local Error',
```

Comment:
this can't work because we `return Promise.reject('Remote Error')`. We should probably `return Promise.reject(new Error('Remote Error'))`